hack/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eux
+
+# Set the release version variable
+OSDK_RELEASE_VERSION=v0.12.0
```

Comment:
this is also defined in the Makefile. it should be defined only once. So bringing the check in this file would solve this issue